.mega-linter.yml

```diff
@@ -0,0 +1,15 @@
+# Configuration file for Mega-Linter
+# See all available variables at https://nvuillam.github.io/mega-linter/configuration/ and in linters documentation
+
+APPLY_FIXES: all # all, none, or list of linter keys
+DEFAULT_BRANCH: main
+SHOW_ELAPSED_TIME: true
+ENABLE:
+  - MARKDOWN
+  # - SPELL Currently 555 errors but would be nice to enable!
+DISABLE_LINTERS:
+  - MARKDOWN_MARKDOWN_TABLE_FORMATTER   # Does not correctly support GFM
+
+# Linter specific settings
```

Comment:
A couple ideas for settings we can put here to alleviate the markdown-link-checker issues:

- `MARKDOWN_MARKDOWN_LINK_CHECK_DISABLE_ERRORS: true` - show markdown link errors as warnings
- `MARKDOWN_MARKDOWN_LINK_CHECK_DISABLE_ERRORS_IF_LESS_THAN: 2` - we can set it to only error if we get more than 2 errors
- Configure `.markdownlinkcheck.json` to ignore all external links and only validate internal links

Reply:
Can't even get a clean CI build due to the link checker :(

Reply:
I think setting them to less than 2 errors seems very sensible - the url that your PR is blocked on works when navigating directly to it in the browser https://www.softwaretestinghelp.com/what-is-integration-testing/ so it's being blocked from being tested programmatically.

Reply:
Yeah I think that might be a good way to reduce the intermittency there, though it could potentially still let things leak through if the links we expect to fail actually pass and then an internal link is broken! Tradeoffs :) Happy to go either way

Reply:
Please merge in the current main branch to fix the link that fails

Reply:
done, and passing!